source/manual/alerts/nginx-429-too-many-requests.html.md

```diff
@@ -32,3 +32,4 @@ You can force creation by creating a zero-value `http_429` counter:
     fab $environment -H cache-1.router statsd.create_counter:cache-1_router.nginx_logs.www-origin.http_429
 
 
+[Nginx logs in Kibana]: https://kibana.logit.io/app/kibana#/discover?_g=()&_a=(columns:!(request,status),filters:!(('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'*-*',key:status,negate:!f,value:'429'),query:(match:(status:(query:429,type:phrase))))),index:'*-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'*')),sort:!('@timestamp',desc))
```

Comment:
It's probably better to just include the query, as the environment this will link to isn't clear (I think it might be linked to whatever environment the person who is logged in to Logit last used).